cuda/factorization/par_ilu_kernels.cu

```diff
@@ -5,12 +5,14 @@
 #include "core/factorization/par_ilu_kernels.hpp"
 
 
+#include <ginkgo/core/base/std_extensions.hpp>
```

Comment:
why is this necessary?

Reply:
probably isn't, again consistency between CUDA and HIP